lib/mgmt_be_client.c

```diff
@@ -323,6 +323,9 @@ static int __send_notification(struct mgmt_be_client *client, const char *xpath,
 	int ret = 0;
 
 	assert(op != NOTIFY_OP_NOTIFICATION || xpath || tree);
+	if (!xpath && !tree){
```

Comment:
This is not correct. Due to the `mgmt_msg_native_xpath_encode` call below, `xpath` actually must be non-NULL. The assert seems to be in error.